base/applications/mspaint/lang/it-IT.rc

```diff
@@ -110,15 +111,15 @@ BEGIN
         MENUITEM "&Copia\tCtrl+C", IDM_EDITCOPY
         MENUITEM "I&ncolla\tCtrl+V", IDM_EDITPASTE
         MENUITEM SEPARATOR
-        MENUITEM "Crop &selection", IDM_CROPSELECTION
+        MENUITEM "Ritaglia &selezione", IDM_CROPSELECTION
         MENUITEM "Cance&lla area selezionata\tCanc", IDM_EDITDELETESELECTION
         MENUITEM "Sele&ziona tutto\tCtrl+A", IDM_EDITSELECTALL
         MENUITEM SEPARATOR
         MENUITEM "Copia &su...", IDM_EDITCOPYTO
         MENUITEM "Incolla &da...", IDM_EDITPASTEFROM
         MENUITEM SEPARATOR
         MENUITEM "Ca&povolgi/ruota...\tCtrl+R", IDM_IMAGEROTATEMIRROR
-        MENUITEM "S&tretch/Skew...\tCtrl+W", IDM_IMAGESTRETCHSKEW
+        MENUITEM "Estendi/Allarha...\tCtrl+W", IDM_IMAGESTRETCHSKEW
```

Comment:
do not strip the accelerators here. check instead which one your it-IT windows 2003 server does use, and add the correct one back.

Reply:
added.